qemu/tests/cfg/hotplug_unplug_during_io_repeat.cfg

```diff
@@ -0,0 +1,27 @@
+- hotplug_unplug_during_io_repeat:
+    no RHEL.3.9
+    no ide
+    virt_test_type = qemu
+    type = hotplug_unplug_during_io_repeat
+    images += " stg0"
+    boot_drive_stg0 = no
+    image_name_stg0 = images/storage0
+    image_size_stg0 = 30G
+    remove_image_stg0 = yes
+    force_create_image_stg0 = yes
+    kill_vm_on_error = yes
+    iozone_timeout = 7200
+    sleep_time = 30
+    repeat_time = 100
+    vt_ulimit_nofile = 10240
+    q35:
+        pcie_extra_root_port = 1
+    Linux:
+        iozone_options = '-az -g 10G -y 32k -i 0 -i 1 -I -f {0}/iozone_test'
+    Windows:
+        iozone_options = '-az -b C:\{0}_stress_test -g 10G -y 32k -i 0 -i 1 -I -f {0}:\iozone_test'
+    variants:
+        - @default:
+        - with_scsi_hba:
```

Comment:
For virtio-blk-pci, windows  cannot support unplug controller when running iozone.

Reply:
> For virtio-blk-pci, windows cannot support unplug controller when running iozone.
Yes, so it is only for virtio_scsi here.

Reply:
Thanks, will update it.

Reply:
unplug device in use scsi-cd/hd/xxx is valid, virtio-scsi-pci is not valid for windows guest